.babelrc

```diff
@@ -3,12 +3,9 @@
     "@babel/typescript",
     "@babel/react",
     ["@babel/env", {
-      "targets": {
-        "chrome": "64"
-      },
-      "useBuiltIns": "usage",
       "debug": false,
-      "corejs": 2
+      "corejs": 3,
+      "useBuiltIns": "entry"
```

Comment:
This may break polyfills for `Array.prototype.flat/flatMap` , which is hard to notice.
Or maybe not, but bad if it happens again